packages/components/src/input-control/types.ts

```diff
@@ -174,6 +174,10 @@ export interface InputBaseProps extends BaseProps, FlexProps {
 	 * If this property is added, a label will be generated using label property as the content.
 	 */
 	label?: ReactNode;
+	/**
+	 * If this property is added, it will be added to the modal frame `div`.
+	 */
+	style?: CSSProperties;
```

Comment:
Instead of adding `style` as an explicit prop, could we use the `WordPressComponentProps` utility type instead for the `InputControl` component? We may be also able to remove other attributes that exist on HTML elements like `className`, `id` and (maybe) `disabled` ?

Reply:
I'll happily update the type to use `WordPressComponentProps` if we decide to go ahead with allowing the styles on the outer wrapper.

Reply:
This has been cleaned up. ( https://github.com/WordPress/gutenberg/pull/45340/commits/fb4df4f4e9d553385ed87f7ad17cfc628471d884 )